NYPL/fedora_ingest_rails
# [DR-1963] Updating list of single value fields

## Jira Tickets ##
[Copy only the first reported value from mms identifier fields](https://jira.nypl.org/browse/DR-1963)

## Things Done ##
- Updated list of solr fields to ensure index as single values. 

## How to Test ##
- Run unit test. 
- Get any document from `/exports/repo_solr_docs/[:uuid]` and convert via the MMSClient `convert_to_solr_docs` method. (E.g., `test_string = "[{\"uuid\":\"8fcbf960-fed9-0130-73f0-58d385a7bbd0\",\"firstInSequence\":[\"8fcbf960-fed9-0130-73f0-58d385a7bbd0\"],\"imageID\":[\"onlyKeepFirst_imageID\",\"getRidOfSecond_imageID\"]..."` and run against `@mms_client.convert_to_json_docs(test_string)`) 
- Expected behavior is for any of the newly-added identifier fields to keep only one value. 

## Questions ##
- This is really overkill, and was intended to be overkill. The values sent should not ever be multivalued. Is this worth doing? I'm not sure. I say yes, just to be on the safe side. Best not to trust strange exports. But I'm open to counsel.
